tests/pr_test.go

```diff
@@ -120,3 +137,45 @@ func TestRunAdvanceExample(t *testing.T) {
 	err := options.RunSchematicTest()
 	assert.Nil(t, err, "This should not have errored")
 }
+
+// helper function to create a valid self-signed TLS cert for https server
+// inspriation from this example: https://go.dev/src/crypto/tls/generate_cert.go
+// outputs: cert, error
+func genNewTlsCert() (string, error) {
```

Comment:
Why don't we just use secrets manager like we did in terraform-ibm-key-protect-key ?

Reply:
moved kmip to advanced example and switched over to the same test we use in kms-key module